hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
Is this good? I ask because attributes can only be a string, number, function, null/undefined, an object in the case of key=`style`.